### PR TITLE
[strings] fi update

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -14112,7 +14173,7 @@
     de = Wasserfläche
     el = Νερό
     es = Cuenca
-    et = Vesi
+    et = Vesiallas
```
Comment (biodranik) [inline data/strings/types_strings.txt:14176]: Should it be changed for fi?

**Review by biodranik — COMMENTED** [review]
Thank you! Can you please double-check that category translations in types_strings.txt are matching categories in data/cagegories.txt? If they are not, then the search won't work properly.

File: data/strings/types_strings.txt (hunk 1 above)
Comment (scop) [reply data/strings/types_strings.txt]: Yes! Thanks for the catch :heart:

**Comment by scop** [comment]
Ah, bummer. Should the cross check be done for strings that were changed in types_strings.txt, or also ones that were added? When there's a mismatch, the one in categories.txt should be changed to match the updated or added one in types_strings.txt?

**Comment by biodranik** [comment]
We plan to make it more convenient. But now they should match. E.g. categories.txt should have at least one, equal translation from types_strings.txt. Also, categories.txt may contain some additional synonyms, which users may potentially type (or mistype) if they want to find the same category of objects.

**Pull request merged**